spec/spec_helper.rb

```diff
@@ -12,13 +12,21 @@
 require 'json'
 
 module Hawkular::Metrics::RSpec
+  MOCKER_VERSION = '0.9.0.Final'
```

Comment:
I did this to avoid re-recording the cassettes, this also allows me to switch versions in the tests in order to test this PR.